pkg/pillar/zedbox/zedbox.go

```diff
@@ -78,31 +78,31 @@ var (
 		"client":           {f: client.Run, inline: inlineAlways},
 		"command":          {f: command.Run},
 		"diag":             {f: diag.Run, inline: inlineUnlessService},
-		"domainmgr":        {f: domainmgr.Run},
-		"downloader":       {f: downloader.Run},
+		"domainmgr":        {f: domainmgr.Run, inline: inlineUnlessService},
```

Comment:
The system should work fine without all of these changes to inlineUnlessService as long as zedbox is started before the other services.